src/keystore/providers/NetworkKeyManager.ts

```diff
@@ -0,0 +1,207 @@
+import { utils } from 'ethers'
```

Comment:
This is borrowed from `store/EncryptedStore`. That will be deleted in the next PR